src/styles/card.scss

```diff
@@ -0,0 +1,136 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-width;
+
+  // height: 408px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  justify-content: center;
+
+  &__image {
+    display: block;
+
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    display: block;
+    height: 36px;
+    font-family: Roboto, sans-serif;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-main-color;
+    box-sizing: border-box;
+    margin: 0 0 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The transform scale on hover is set to 1.2, which might be too large and could cause layout shifts. Consider using a smaller scale value to prevent disrupting the layout.